tests/cargo-add.rs

```diff
@@ -1,7 +1,6 @@
 extern crate assert_cli;
 #[macro_use]
 extern crate pretty_assertions;
-extern crate tempdir;
```

Comment:
You've deleted `extern crate tempdir`, because it is no longer used directly, only through `utils::clone_out_test`, right?
How about `tests/cargo-upgrade.rs`?

Reply:
We use `tempdir` directly in the `upgrade_workspace` test to create a temporary directory to contain the workspace to upgrade.

Reply:
Oh, never mind, I was looking at my old local version.